app/src/main/java/org/wikipedia/dataclient/wikidata/Claims.kt

```diff
@@ -0,0 +1,101 @@
+package org.wikipedia.dataclient.wikidata
+
+import android.location.Location
+import com.google.gson.JsonElement
+import com.google.gson.annotations.SerializedName
+import org.wikipedia.dataclient.mwapi.MwResponse
+import org.wikipedia.json.GsonUtil
+
+class Claims : MwResponse() {
+
+    private val claims: Map<String, List<Claim>>? = null
+
+    fun claims(): Map<String, List<Claim>> {
+        return claims ?: emptyMap()
+    }
```

Comment:
This can be one line.

`private val claims: Map<String, List<Claim>> = emptyMap()`